app/src/main/java/ui/settings/about/PersonAdapter.kt

```diff
@@ -109,80 +75,92 @@ class PersonAdapter internal constructor() : BaseRecyclerViewAdapter(), KoinComp
         return HeaderHolder(textView)
     }
 
-    override fun getItemViewType(position: Int): Int = if (items[position] is Person) PERSON else -1
+    override fun getItemViewType(position: Int): Int =
+        if (items[position] is String) VIEW_TYPE_PERSON else -1
 
     override fun getItemCount(): Int = items.size
 
     /**
      * Header in the list
      */
     internal inner class HeaderHolder(private val view: TextView) :
-            BaseRecyclerViewAdapter.BaseHolder(view) {
+        BaseRecyclerViewAdapter.BaseHolder(view) {
 
         override fun bind(position: Int) {
-            view.setText(items[position] as Int)
+            val title = items[position] as? Int ?: return
+            view.setText(title)
         }
     }
 
     /**
      * Person item in the list
      */
     internal inner class PersonHolder(parent: ViewGroup) :
-            BaseRecyclerViewAdapter.BaseHolder(parent, R.layout.item_person) {
+        BaseRecyclerViewAdapter.BaseHolder(parent, R.layout.item_person) {
 
         override fun bind(position: Int) {
-            val person = items[position] as Person
+            val personName = items[position] as? String ?: return
+
+            val idPrefix = "about_$personName"
 
             itemView.apply {
-                name.setText(person.name)
+                // Name
+                val nameId = getResourceId(idPrefix)
+                if (nameId != 0) {
+                    name.setText(nameId)
+                } else {
+                    name.text = ""
+                }
 
-                Picasso.get()
-                        .load(person.pictureId)
+                // Picture
+                val pictureId = getResourceId(idPrefix, "drawable")
+                if (pictureId != 0) {
+                    Picasso.get()
+                        .load(pictureId)
                         .into(picture)
+                } else {
+                    picture.visibility = View.GONE
+                }
 
-                role.setText(person.role)
-
-                linkedIn.setOnClickListener {
-                    ga.sendEvent("About", "Linkedin", context.getString(person.name))
-                    context.openUrl(context.getString(person.linkedIn))
+                // LinkedIn
+                val linkedInId = getResourceId(idPrefix + "_linkedin")
```

Comment:
For the most part, it's recommended that you don't concatenate strings. You'll probably get a warning about this in lint as well. In Kotlin, wrap variables with `${...}`. So this becomes `${idPrefix}_linkedin`. In cases where  only a variable is wrapped and the proceeding characters aren't letters, you. Ktlint should actually do that automatically when it's added

Reply:
Weird, I normally don't do this. I must be tired...